CHANGES.md

```diff
@@ -32,6 +32,8 @@
 - `.pytest_cache`, `.ruff_cache` and `.vscode` are now excluded by default (#3691)
 - Fix black not honouring `pyproject.toml` settings when running `--stdin-filename` and
   the `pyproject.toml` found isn't in the current working directory (#3719)
+- Black will now error if `exclude` and `extend-exclude` have invalid data types in
+  `pyproject.toml`, instead of silently doing the wrong thing.
```

Comment:
```suggestion
  `pyproject.toml`, instead of silently doing the wrong thing. (#3764)
```

Reply:
Github is doing something really weird where the pushes show on the branch in my fork, but don't get reflected in the PR https://github.com/hauntsaninja/black/commit/9f55e735ddbe600565e033543208689787b4b7db

Reply:
Yeah seems like there is something weird going on. In any case, this PR looks good once GitHub cooperates.

Reply:
For whatever reason, the force in force push seems to wake it up. So I just need to commit an empty commit, push it, delete the commit and force push.

Reply:
Sounds like a great workflow